internal/pkg/scaffold/controller_kind.go

```diff
@@ -36,29 +43,44 @@ func (s *ControllerKind) GetInput() (input.Input, error) {
 	// Error if this file exists.
 	s.IfExistsAction = input.Error
 	s.TemplateBody = controllerKindTemplate
+
+	// Set imports.
+	if len(s.ImportMap) == 0 {
```

Comment:
When would this not be empty?

Reply:
Yeah even if this exported we should treat it like an unexported field and always overwrite it, good call. This needs to be exported for templating purposes.